rviz_voxelgrid_visuals/src/voxel_visual.cpp

```diff
@@ -81,7 +81,12 @@ void DenseVoxelGridVisual::updatePointCloud() {
           val = 1.0;
         }
 
-        p.setColor(r_, g_, b_, std::min(val * a_, 1.0f));
+        if (latest_msg.has_color) {
+          p.setColor(latest_msg.color.r, latest_msg.color.g, latest_msg.color.b,
+                     std::min(val * latest_msg.color.a, 1.0f));
```

Comment:
`std::min(val * latest_msg.color.a * a_, 1.0f));`

Might as well allow the "Alpha Multiple" field to still affect things